Lib/ntpath.py

```diff
@@ -763,9 +760,9 @@ def realpath(path, *, strict=False):
             # For UNC paths, the prefix will actually be \\?\UNC\
             # Handle that case as well.
             if path.startswith(unc_prefix):
-                spath = new_unc_prefix + path[len(unc_prefix):]
+                spath = new_unc_prefix + path[8:]
             else:
-                spath = path[len(prefix):]
+                spath = path[4:]
```

Comment:
How much does this specific change speed thjings up? The `len()` calls are obviously slightly inefficient, but the advantage is that it's _really_ obvious for the reader what the significance of the slice starting point is. The change here makes things a little less readable, unfortunately, and also more fragile -- there's a risk that future updates to this function might accidentally break, because the person rewriting the function doesn't realise that they also need to update the numbers `8` and `4` here

Reply:
Probably not a lot. I've reverted it for the reasons stated.